### PR TITLE
Updates for TRT-LLM 0.9

### PR DESCRIPTION
**Pull request by oyilmaz-nvidia**
# What does this PR do ?

Updates export to TRT-LLM code for version 0.9
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation
- [x] Update

### CODE REVIEW

File: nemo/export/trt_llm/decoder/gpt.py [hunk 1]
```diff
@@ -85,37 +86,67 @@ class GPTDecoderLayerBuilder(DecoderLayerBuilder):
...
+        config.set_if_not_exist('hidden_act', self.hidden_act)
+        config.set_if_not_exist('apply_query_key_layer_scaling', False)
+        config.set_if_not_exist('bias', bias_qkv)
+        config.set_if_not_exist('rotary_base', layer.rotary_base)
+        config.set_if_not_exist('rotary_scaling', rotary_scaling)
+        config.set_if_not_exist('rotary_pct', rotary_pct)
+        config.set_if_not_exist('moe_num_experts', 0)
+
+        '''
+                return GPTDecoderLayer(
```
Comment (JimmyZhang12) [inline nemo/export/trt_llm/decoder/gpt.py:130]: can this comment be deleted?

File: nemo/export/trt_llm/utils.py [hunk 2]
```diff
@@ -19,6 +19,8 @@
 import tarfile
 import tempfile
 import typing
+from pathlib import Path
```
Comment (JimmyZhang12) [inline nemo/export/trt_llm/utils.py:22]: Seems this is being imported but not used?

**Review by JimmyZhang12 — APPROVED** [review]
Just left 2 minor comments, lgtm!

**Comment by oyilmaz-nvidia** [comment]
jenkins

**Review by ericharper — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**